MPK01/MPK01.02/lvfs/com.logitech.MPK01.metainfo.xml

```diff
@@ -15,7 +15,7 @@
   </provides>
   <url type="homepage">http://www.logitech.com/en-us/product/k780-multi-device-wireless-keyboard</url>
   <metadata_license>CC0-1.0</metadata_license>
-  <project_license>Proprietary</project_license>
+  <project_license>proprietary</project_license>
```

Comment:
I guess this is right, but you might as well use the SPDX `LicenseRef-proprietary` instead if you want to me super clear.